uktrade/data-hub-frontend
# Add Task logic for investment opportunities

## Description of change

Adds the Task Logic for Investment Opportunities and displays the Opportunity Details. I've hidden the Opportunity Requirements and form edit toggling for now to keep this a reasonably-sized PR - will make a PR for the Opportunity Requirements after this one. (There are a couple of places where I've ended up starting the abstractions that will be used when the Requirements are added.

## Test instructions
Either view the page locally at: http://localhost:3000/investments/opportunities/5d341b34-1fc8-4638-b4b1-a0922ebf401e/details (any uuid should work)

Or if running the latest version of Data Hub API: view at:
http://localhost:3000/investments/opportunities/c064e66a-a6bc-4ad6-9a88-68502d15499c/details
http://localhost:3000/investments/opportunities/744cd12a-6095-e211-a939-e4115bead28a/details

## Screenshots
<img width="1094" alt="Screenshot 2021-03-29 at 14 57 21" src="https://user-images.githubusercontent.com/23265724/112850087-98a4be80-90a1-11eb-9f81-e71971242f04.png">


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
